libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/Color.kt

```diff
@@ -25,7 +25,7 @@ val LightGrey = Color(0x993C3C43)
 @ShowkaseColor(name = "DarkGrey", group = "Material Design")
 val DarkGrey = Color(0x99EBEBF5)
 
-val AvatarGradientStart = Color(0xFF4CA1AF)
+val AvatarGradientStart = Color.Red
```

Comment:
Do not forget to revert this change :)

Reply:
I'll do 1 more change to try to simplify the script a bit more and then revert this change if it works 👍

Reply:
Done!